force_wfmanager/left_side_pane/variable_names_register.py

```diff
@@ -0,0 +1,63 @@
+from traits.api import HasStrictTraits, List, Instance, on_trait_change
+
+from force_bdss.api import Identifier
+
+from .mco_parameter_model_view import MCOParameterModelView
+from .data_source_model_view import DataSourceModelView
+from .kpi_calculator_model_view import KPICalculatorModelView
+
+
+class VariableNamesRegister(HasStrictTraits):
```

Comment:
please add some comment on the purpose of the class and its interaction with the others because it's not immediate.